modules/local/rosettafold2na_fasta/main.nf

```diff
@@ -0,0 +1,145 @@
+process ROSETTAFOLD2NA_FASTA {
+    tag   "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.8.3' :
+        'biocontainers/python:3.8.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("rf2na_input", type: "dir"), emit: prepared_input
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    """
+    python3 - <<'PY' "${meta.id}" "${fasta}"
```

Comment:
For using the script this way and given it is quite long, I will put it in the bin folder instead and only call it here:
```
fasta2rosettafold.py "${meta.id}" "${fasta}"
```

Actually, probably we should join the different scripts that perform this for different modes in a single one in bin, but this will be for a follow-up PR